CHANGELOG.md

```diff
@@ -25,6 +23,9 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
 - `ngl.get_backends()` and `ngl.probe_backends()` were mistakenly inverted in
   v2023.2, this is changed back in this release
 - `SceneCfg` and `SceneInfo` now use the `Backend` enum instead of a string
+- `ngl_configure()` does not update the `backend` field from the user
+  configuration anymore and the `config` argument is now const. Users must now
+  use `ngl_get_backend()` to get the underling backend information
```

Comment:
underlying (underling refers to persons iiuc)

Reply:
Fixed.